akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala

```diff
@@ -24,8 +25,8 @@ object BackoffSupervisor {
    *   started again, if it is terminated
    * @param maxBackoff the exponential back-off is capped to this duration
    * @param randomFactor after calculation of the exponential back-off an additional
-   *   random delay based on this factor is added, e.g. 0.2 adds up to 20%
-   *   delay
+   *   random delay based on this factor is added, e.g. `0.2` adds up to `20%` delay.
```

Comment:
it is highly encouraged to add randomization. This should be mentioned in the docs.